prometheus/rootfs/etc/cont-init.d/prometheus.sh

```diff
@@ -6,5 +6,6 @@
 echo "${SUPERVISOR_TOKEN}" > '/run/home-assistant.token'
 
 mkdir -p /data/prometheus
+chown -R prometheus:prometheus /data/prometheus
 find /data/prometheus -not -perm 0644 -type f -exec chmod 0660 {} \;
 find /data/prometheus -not -perm 0755 -type d -exec chmod 0770 {} \;
```

Comment:
We should migrate the add-on to run as root instead (as per the general design of Home Assistant).

Reply:
Yes, that would probably eliminate the need for checking the ownership and permissions of the /data/prometheus/* files. Did you do the original work on this addon? I may take a stab at converting it to run as root, but not sure when I can get to it.

Thanks for working with me on my very first HA PR.

-Nathan